includes/admin/plugins.php

```diff
@@ -142,8 +142,8 @@ function perflab_render_plugin_card( array $plugin_data ) {
 	$version     = $plugin_data['version'];
 	$name        = wp_strip_all_tags( $title . ' ' . $version );
 
-	$requires_php = isset( $plugin_data['requires_php'] ) ? $plugin_data['requires_php'] : null;
-	$requires_wp  = isset( $plugin_data['requires'] ) ? $plugin_data['requires'] : null;
+	$requires_php = $plugin_data['requires_php'] ?? null;
+	$requires_wp  = $plugin_data['requires'] ?? null;
```

Comment:
Not blocker: WordPress code did not support it at the moment so we can do this changes in future when WP officially support it.

Reply:
Yes, some code utilizes it, but let's check some discussions regarding it.

- https://core.trac.wordpress.org/ticket/58874
- https://wordpress.slack.com/archives/C02RQBWTW/p1695705092378319?thread_ts=1695704573.754599&cid=C02RQBWTW
- https://github.com/WordPress/wordpress-develop/pull/5302

Reply:
It seems the main concern is back porting to old WP versions. Given that is not relevant here for new Performance features, I say we keep it.

Reply:
Ok. Let's keep it then.